test/api-annotation-document.test.js

```diff
@@ -32,8 +32,8 @@ describe('<api-annotation-document>', () => {
       await nextFrame();
     });
 
-    it('is accessible', async () => {
-      await assert.isAccessible(element);
```

Comment:
The `assert.isAccessible()` is an asynchronous function @TwoPlusTwoOne I made it myself :) However, folks from the  open-wc did not create proper types for them.